app/core/src/main/java/com/fsck/k9/preferences/SettingsFileParser.kt

```diff
@@ -1,414 +1,441 @@
-package com.fsck.k9.preferences;
```

Comment:
I think we should split the Java to Kotlin conversion from the rafactoring into separate commits. With both mixed it is really hard to follow up on the changes.